database_operations.py

```diff
@@ -104,4 +108,50 @@ def delete_notified_reminders():
         logger.error(f"Se produjo un error al eliminar recordatorios notificados: {e}", exc_info=True)
     finally:
         session.close()
-        
\ No newline at end of file
+
+class AnalisisButton(discord.ui.Button):
+    def __init__(self, message_id, channel_id):
```

Comment:
The channel_id is necessary as it indicates the target channel where the thread will be created.